source/logging/README.md

```diff
@@ -0,0 +1,69 @@
+# Logging
+ **Notice:** Running the AWS IoT Device Client will incur usage of AWS IoT services, and is likely to incur charges on your AWS account. Please refer the pricing pages for [AWS IoT Core](https://aws.amazon.com/iot-core/pricing/), [AWS IoT Device Management](https://aws.amazon.com/iot-device-management/pricing/), and [AWS IoT Device Defender](https://aws.amazon.com/iot-device-defender/pricing/) for more details.
+
+ * [Logging](#logging)
+    + [Logging Configuration Options](#logging-configuration-options)
+      - [Configuring the logger via the command line](#configuring-the-logger-via-the-command-line)
+      - [Configuring SDK logging via the command line](#configuring-sdk-logging-via-the-command-line)
+      - [Configuring the logger via the JSON configuration file](#configuring-the-logger-via-the-json-configuration-file)
+      - [Configuring SDK logging via the JSON configuration file](#configuring-sdk-logging-via-the-json-configuration-file)
+
+[*Back To The Main Readme*](../../README.md)
+
+## Logging
+The AWS IoT Device Client has the capability to log directly to standard output or write logs to a log file. For
+either option, the logging level can be specified as either DEBUG, INFO, WARN, or ERROR. The logger implementation
+can be specified as either "STDOUT" (for standard output) or "FILE" (for logging to a file). If file based logging
+is specified, you can also specify a file to log to. If a file is not specified, the Device Client will log to 
+the default log location of `/var/log/aws-iot-device-client/aws-iot-device-client.log`. Keep in mind that the Device Client will need
+elevated permissions to log to this location, and will automatically fall back to STDOUT logging if the Device Client
+is unable to log to either the specified or default location. 
+
+The AWS IoT Device Client also provides the ability to enable native SDK logging from the AWS Common Runtime (CRT). By
+default, enabling this functionality will log TRACE level logs and above to `/var/log/aws-iot-device-client/sdk.log`
+but these defaults can be overridden as show below. If no configuration options are passed, SDK logging is disabled
+by default.
+
+### Logging Configuration Options
+
+#### Configuring the logger via the command line
+```
```

Comment:
In addition to the examples, I think we should add small/quick descriptions for each option. Some are pretty clear but it won't hurt to add quick description and list of allowed values for each of them, but some are a little bit vague like `log-type` for example begs the question what are the other possible types? how to use them ..etc.

Reply:
The options are mentioned in the first paragraph

Reply:
Ah, I missed that, it's probably better to use the format used in other README files, for example [here](https://github.com/awslabs/aws-iot-device-client/tree/206759a6264a23a52541867580d0953bc41d8541/source/devicedefender#device-defender-feature-configuration-options) but I guess it's fine since it's in the description.